src/config/mod.rs

```diff
@@ -105,17 +114,24 @@ impl Config {
                     theme: custom.theme.unwrap_or(default.theme),
                     remote_styles: custom.remote_styles.unwrap_or(default.remote_styles),
                     additional_css: custom.additional_css.unwrap_or(default.additional_css),
-                    repository: custom.repository,
+                    repository: Some(
+                        custom
+                            .repository
+                            .unwrap_or_else(|| project.repository.unwrap_or_default()),
+                    ),
```

Comment:
hmm do you really want to end up with `Some("")` if nothing is set? Why not let it be an actual Option?

```
repository: custom.respository.or(project.repository)
```

Reply:
yeah you are super right, changed